packages/expo-cli/src/commands/publish-info.ts

```diff
@@ -46,52 +30,10 @@ export default (program: any) => {
       parseInt
     )
     .option('-p, --platform <ios|android>', 'Filter by platform, android or ios.')
+    .option('-s, --sdk-version <version>', 'Filter by sdk version e.g. 35.0.0')
```

Comment:
```suggestion
    .option('-s, --sdk-version <version>', 'Filter by SDK version e.g. 35.0.0')
```

also maybe we should support `35` or `36` instead of fully qualified .0.0 version